package.json

```diff
@@ -19,7 +19,8 @@
   },
   "peerDependencies": {
     "@nestjs/common": "^9.0.0",
-    "@nestjs/core": "^9.0.0"
+    "@nestjs/core": "^9.0.0",
+    "express": "^4.8.1"
```

Comment:
`express` is optional here. You could add `fastify` as well and mark both of them as optional with https://docs.npmjs.com/cli/v8/configuring-npm/package-json#peerdependenciesmeta

Reply:
Great idea. I'll do that and test it out with yarn pnp workspaces.

Reply:
@micalevisk I updated with the following:
* I had the wrong express version, so fixed that.
* Added fastify and fastify/startic.
* Added all 3 to peerDependenciesMeta so they are optional